k8s/go/cmd/resolver/resolver.go

```diff
@@ -37,6 +37,7 @@ var (
 	k8sTemplate       = flag.String("template", "", "The k8s YAML template file to resolve.")
 	substitutionsFile = flag.String("substitutions", "", "A file with a list of substitutions that were made in the YAML template. Any stamp values that appear are stamped by the resolver.")
 	allowUnusedImages = flag.Bool("allow_unused_images", false, "Allow images that don't appear in the JSON. This is useful when generating multiple SKUs of a k8s_object, only some of which use a particular image.")
+	noPush            = flag.Bool("no_push", false, "Don't push images after resolving digests.")
```

Comment:
Will you avoid the `no` prefix?  Google code tends to consider `--nopush` equivalent to `--push-false` and is therefore unnecessarily confusing (even with the underscore).

```golang
flag.Bool("skip_push", false, "...")
flag.Bool("push", true, "...")
```

Reply:
Send a follow-up for this?